gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -746,14 +746,14 @@ message TripDescriptor {
   // direction of travel for trips this selector refers to.
   optional uint32 direction_id = 6;
 
-  // The initially scheduled start time of this trip instance.
```

Comment:
I'm intentionally trying not to use the word "scheduled" here because the `TripDescriptor` can refer to both scheduled (normal GTFS without frequencies.txt, frequencies.txt exact_times=1) and frequency-based (frequencies.txt exact_times=0) trips. I'm currently proposing that we change the text to just:

>The start time of this trip instance.

...and let the remainder of the paragraph describe the conditions under which it's scheduled or frequency-based.

IMHO it's already confusing enough that we have scheduled trips represented in frequencies.txt with exact_times=1 trips, so I'd like to be more precise in the language and not use "schedule" when referring to exact_times=0 trips.